### PR TITLE
Cloud changelog - Feb

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,48 @@
...
-- [Release 2021-11-11](#release-2021-11-11)
+
+## Release 2022-02-02
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated prepackaged Boards version to 0.12.1.
+ - Added Boards support for Global Retention Policy.
+ - Added a Collapsed Reply Threads tour functionality.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:20]:
```suggestion
 - Added Collapsed Reply Threads (Beta) tour functionality.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,48 @@
...
+ - Added the current product name information to invoices.
+ - Added support for metered billing invoices.
+ - Added a keyboard shortcut to open the expanded right-hand side.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:23]:
```suggestion
 - Added a keyboard shortcut to open and expand the right-hand pane.
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -3,12 +3,48 @@
...
+
+#### Bug Fixes
+ - Fixed an issue where the user menu header was visible when custom statuses was disabled.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:26]:
```suggestion
 - Fixed an issue where the user menu header was visible when custom statuses were disabled.
```

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -3,12 +3,48 @@
...
+ - Fixed an issue where the markdown preview link was not hidden in read-only channels.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:27]:
```suggestion
 - Fixed an issue where the markdown **Preview** link was not hidden in read-only channels.
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -3,12 +3,48 @@
...
+ - Fixed an issue where Cloud admins were unable to download an invoice via the Desktop App.
+ - Fixed an issue that caused a gap to appear on the left-hand side in products using the team sidebar.
+ - Fixed an issue with Collapsed Reply Threads where clearing a deleted root post left the right-hand side blank.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:30]:
```suggestion
 - Fixed an issue with Collapsed Reply Threads (Beta) where clearing a deleted root post left the right-hand side blank.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -3,12 +3,48 @@
...
+ - File upload might fail for SVG files [MM-38982](https://mattermost.atlassian.net/browse/MM-38982).
+ - ``CMD+/`` does not close the shortcuts modal [MM-38971](https://mattermost.atlassian.net/browse/MM-38971).
+ - ``CTRL/CMD + SHIFT + A`` shortcut does not open **Settings** [MM-38236](https://mattermost.atlassian.net/browse/MM-38236).
+ - Known issues related to the new Collapsed Reply Threads (Beta) are [listed here](https://docs.mattermost.com/messaging/organizing-conversations.html#known-issues).
+ - System Roles shows **License** and **Environment** as possible permissions, but they are always hidden in Cloud.
+
+## Release 2022-01-27
+
+### Bug Fixes
+ - Fixed an issue where replying to posts, pinning posts and editing posts did not work.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:47]:
```suggestion
 - Fixed an issue where replying to posts, pinning posts, and editing posts did not work.
```

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline.

**Review by esethna — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Pull request merged**